engine/src/exec-interface2.cpp

```diff
@@ -1988,7 +1988,9 @@ void MCInterfaceSetRelayerGroupedControls(MCExecContext& ctxt, bool p_value)
 void MCInterfaceSetBrush(MCExecContext& ctxt, Properties p_which, uinteger_t p_value)
 {
 	uint4 t_newbrush = p_value;
-	if (t_newbrush < PI_PATTERNS)
+	// MDW-2016-05-21 [[ bugfix_17553 ]] map low brush values up to 101..135 range
+	// if less than 36, map to 101..135
```

Comment:
@mwieder we don't require the author note anymore and rely on the git log instead. These comments could be removed but we would need to fix the range if they aren't removed.